packages/app/src/debug/LayeredBrowserIcons.cy.tsx

```diff
@@ -19,12 +19,13 @@ describe('<LayeredBrowserIcons/>', () => {
   it('mounts correctly for multiple browsers', () => {
     cy.mount(() => (
       <div class='bg-gray-50 p-24px'>
-        <LayeredBrowserIcons browsers={browsers} />
-        <LayeredBrowserIcons browsers={['CHROME', 'FIREFOX', 'EDGE']}/>
-        <LayeredBrowserIcons browsers={['CHROME-CANARY', 'WEBKIT', 'ELECTRON']} />
-        <LayeredBrowserIcons browsers={['FIREFOX', 'WEBKIT', 'EDGE']}/>
-        <LayeredBrowserIcons browsers={['FIREFOX', 'EDGE', 'CHROME-CANARY', 'WEBKIT']}/>
-        <LayeredBrowserIcons browsers={['EDGE', 'CHROME', 'WEBKIT', 'ELECTRON']}/>
+        <LayeredBrowserIcons order={browsers} />
+        <LayeredBrowserIcons order={['CHROME', 'FIREFOX', 'EDGE']}/>
+        <LayeredBrowserIcons order={['CHROME-CANARY', 'WEBKIT', 'ELECTRON']} />
+        <LayeredBrowserIcons order={['FIREFOX', 'WEBKIT', 'EDGE']}/>
+        <LayeredBrowserIcons order={['FIREFOX', 'EDGE', 'CHROME-CANARY', 'WEBKIT']}/>
+        <LayeredBrowserIcons order={['EDGE', 'CHROME', 'WEBKIT', 'ELECTRON']}/>
+        <LayeredBrowserIcons order={['CHROME', 'CHROME-BETA', 'CHROME-CANARY']} />
```

Comment:
Not a part of this PR, but it looks like the [screenshot didn't render](https://percy.io/cypress-io/cypress/builds/25735966/changed/1434750667?browser=chrome&browser_ids=34&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=800&widths=450%2C800) in Percy

Could we add an assertion to this test that would confirm the icons have actually rendered in the DOM before the Percy snapshot happens?